src/crypto/des_3.c

```diff
@@ -0,0 +1,25 @@
+#include "crypto.h"
+
+static const CipherGetter chiphers[] = {
+    EVP_des_ede3_cbc,
+    EVP_des_ede3_ecb,
+    EVP_des_ede3_cfb8,
+    EVP_des_ede3_ofb,
+};
+
+static inline const EVP_CIPHER *get_des_3_cipher(const CIPHER_MODES mode)
+{
+    return chiphers[mode]();
```

Comment:
Corrected spelling of 'chiphers' to 'ciphers'.

```suggestion
    return ciphers[mode]();
```